datascience/tables.py

```diff
@@ -2052,6 +2052,29 @@ def annotate(axis, ticks):
 
         self._visualize(column_for_categories, labels, xticks, overlay, draw, annotate, width=width, height=height)
 
+
+    def group_bar(self, column_label, **vargs):
+        """Plot a bar chart for the table.
+
+        The values of the specified column are grouped and counted, and one
+        bar is produced for each group.
```

Comment:
Okay, I've added a comment in my next commit.  I think this method is actually a good addition to the API, not just for pedagogical purposes.  Its behavior is more intuitive when you're thinking about `bar` as displaying a distribution (like `hist`) rather than a series of points with heights (like `scatter`).  But I agree it will cause confusion to have both methods.  Hopefully the comment will minimize that.